### PR TITLE
ui: fixes part2

### PR DESCRIPTION
**Pull request by MaximilianMeister**

### CODE REVIEW

File: app/views/devise/registrations/new.html.slim [hunk 1]
```diff
@@ -18,7 +18,7 @@
       .form-group
         = f.email_field :email, autofocus: true, class: 'form-control', placeholder: 'Introduce your Email', required: true
       .form-group
-        = f.password_field :password, class: 'form-control', placeholder: 'Select your password (8 characters min.)', required: true
+        = f.password_field :password, class: 'form-control', pattern: "^[a-zA-Z0-9]{#{Devise.password_length.begin},#{Devise.password_length.end}}$", placeholder: "Select your password (#{Devise.password_length.begin} characters min.)", required: true
```
Comment (mssola) [inline app/views/devise/registrations/new.html.slim]: `[a-zA-Z0-9]` Why are other characters not allowed ?

Comment (MaximilianMeister) [inline app/views/devise/registrations/new.html.slim]: no specific reason, that's just a suggestion, should we restrict this or not?

Comment (mssola) [inline app/views/devise/registrations/new.html.slim]: Not in my opinion, but I'd like to hear from others.

Comment (MaximilianMeister) [inline app/views/devise/registrations/new.html.slim]: i think if there's no restriction from the back-end side, it's better to not restrict here either

**Comment by jimmykarily** [comment]
Css is broken:

selection_004

**Comment by jimmykarily** [comment]
Also I get no flash message at all in the registration page (e.g. when I enter 2 different passwords and submit).

File: app/views/devise/registrations/new.html.slim (hunk 1 above)
Comment (ereslibre) [inline app/views/devise/registrations/new.html.slim]: I would not restrict password allowed symbols at all, I am all in for enforcing a minimum length, but characters should be any. Here is an example: http://html5pattern.com/Passwords (`(?=^.{8,}$)((?=.*\d)|(?=.*\W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*$`). Something similar to that?

**Comment by MaximilianMeister** [comment]
> Css is broken

fixed

**Comment by jimmykarily** [comment]
selection_006

**Pull request merged**